src/libstd/rand/os.rs

```diff
@@ -30,13 +30,16 @@ mod imp {
                   target_arch = "x86",
                   target_arch = "arm",
                   target_arch = "aarch64",
-                  target_arch = "powerpc")))]
+                  target_arch = "powerpc",
+                  target_arch = "powerpc64",
+                  target_arch = "powerpc64le")))]
     fn getrandom(buf: &mut [u8]) -> libc::c_long {
         #[cfg(target_arch = "x86_64")]
         const NR_GETRANDOM: libc::c_long = 318;
         #[cfg(target_arch = "x86")]
         const NR_GETRANDOM: libc::c_long = 355;
-        #[cfg(any(target_arch = "arm", target_arch = "powerpc"))]
+        #[cfg(any(target_arch = "arm", target_arch = "powerpc",
+                  target_arch = "powerpc64", target_arch = "powerpc64le"))]
         const NR_GETRANDOM: libc::c_long = 384;
```

Comment:
If I'm reading this right that's saying __NR_getrandom == 384, which is wrong. It's 359 for all powerpc kernels.